portlist/netstat.go

```diff
@@ -2,21 +2,23 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-//go:build !ios && !js
+//go:build darwin && !ios
 
 package portlist
 
 import (
-	"sort"
-	"strconv"
-	"strings"
+	"bufio"
+	"bytes"
+	"io"
+
+	"go4.org/mem"
 )
 
-func parsePort(s string) int {
+func parsePort(s mem.RO) int {
```

Comment:
Not added in this PR, but: any chance of a docstring that explains why not net.SplitHostPort? I believe the answer is because we also have to parse `host.port` format (is that still true now that this is macOS only?).

Reply:
Documented.